camayoc/tests/qpc/conftest.py

```diff
@@ -39,7 +36,6 @@ def sort_and_delete(trash):
             assert response.status_code < 500
 
 
-@pytest.fixture(scope='function')
```

Comment:
Have you tested this change running all tests? I would think that a bunch of tests would have to change because they are all expecting the `cleanup` fixture